src/ke.cpp

```diff
@@ -196,6 +196,11 @@ KeLowerIrql(_In_ KIRQL new_irql)
     ASSERT(result);
 }
 
+void KfLowerIrql(_In_ KIRQL new_irql)
+{
+    return KeLowerIrql(new_irql);

```

Comment:
I'm not sure it matters, but IIRC the NT kernel implements these the other way around, i.e. KeLowerIrql calls KfLowerIrql. It would only matter if performing a stack backtrace or if someone attempted to hook the "lowest" function call.

Reply:
I don't think it matters.